tests/common-reactive/build.gradle

```diff
@@ -3,6 +3,7 @@ plugins {
 }
 
 dependencies {
+    annotationProcessor(mnSerde.micronaut.serde.processor)
```

Comment:
Had to add this with new gradle plugin that I suppose removes or does not explicitly add annotation processor dependency like it did before.